SalesforceCommerceCloud/pwa-kit
# Use SLAS for Guest Login on Server-Side

**GUS**: (https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000Od9uYAC/view)


## How to test-drive this PR
- Check out 
- Run locally an make sure the app is working as expected regarding SLAS tokens
- Logging in logging out etc.

## Changes
- Added methods to create PKCE flow flow code challenge and verifiers on the server and removed the guest login from the shopper API
- Installed one package 'randomstring' and made some changes to webpack config
